baseline.yaml

```
@@ -649,6 +649,27 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-13
+    maturity_level: 1
+    category: Documentation
+    criteria: |
+       The project documentation MUST include a 
+       descriptive statement about the scope and 
+       duration of support. 
+    objective: |
+       Identify and set expectations for downstream 
+       consumers around what updates the project may
+       or may not provide and when no further updates
+       will be provided.
+    implementation: |
+       Create a status check that checks the project's
+       publicly-available collateral for support 
+       and/or lifecycle statements or machine-readable 
+       metadata.
```

The "implementation" text is normally how a *project* implements the requirement, not how external parties can *verify* it.

```suggestion
       Provide text identifying which versions/branches of the project
       are being supported and at what level (e.g., improvements welcome,
       bug fixes only, security fixes only, or abandoned).
       Where practical, provide lifecycle statements or machine-readable 
       metadata, e.g., via security-insights.yml project-lifecycle.status.       
```

FYI, here's more info on [security-insights project-lifecycle](https://github.com/ossf/security-insights-spec/blob/main/specification/project-lifecycle.md), but we haven't agreed on how to include links so I haven't suggested it.

---

Small changes for style. Please double check me here @david-a-wheeler 

```suggestion
	Provide text identifying which iterations of the
	project are being supported, and at what level.
	Some examples include: improvements welcome,
	bug fixes only, security fixes only, or
	abandoned. 

	It is recommended that projects provide lifecycle
	statements or machine-readable  metadata such as
	security-insights.yml project-lifecycle.status.
```